docs/concepts/workloads/controllers/deployment.md

```diff
@@ -346,20 +347,20 @@ nginx-deployment-2035384211) and new replicas (nginx-deployment-3066724191) are
 ```shell
 $ kubectl get rs
 NAME                          DESIRED   CURRENT   READY   AGE
-nginx-deployment-1564180365   2         2         0       25s
-nginx-deployment-2035384211   0         0         0       36s
-nginx-deployment-3066724191   2         2         2       6s
+nginx-deployment-5964dfd755   3         3         3       1m
+nginx-deployment-5d5cfdbd5f   1         1         0       20s
+nginx-deployment-6c54bd5869   0         0         0       3m
 ```
 
-Looking at the Pods created, you will see that the 2 Pods created by new ReplicaSet are stuck in an image pull loop.
+Looking at the Pods created, you will see that the 1 Pods created by new ReplicaSet are stuck in an image pull loop.
```

Comment:
Some grammatical errors. Should be "you will see that the Pod created by the new ReplicaSet is stuck in an image pull loop."

Reply:
@bradtopol Thanks for your review. I forgot to mention that in my testings there is only 1 pod which stucked in an image pull loop (not 2 pods described before).